.github/workflows/ci.yml

```diff
@@ -378,3 +383,23 @@ jobs:
 
       - name: Build
         run: make BASE=./base.tar.gz all
+
+  create_release:
```

Comment:
This should be separate from `ci.yml`, since this file is just for the CI run. Maybe we need a new workflow file for release?

Reply:
I don't think that a separate workflow file makes sense here.  Having a release step in a workflow is a pretty normal step and gating it to only run when a release tag is created is also pretty standard.

Reply:
lets follow up offline on this.